central/detection/service/service_impl.go

```diff
@@ -239,6 +254,15 @@ func (s *serviceImpl) enrichAndDetect(ctx context.Context, enrichmentContext enr
 	}, nil
 }
 
+func (s *serviceImpl) getAppliedNetpolsForDeployment(ctx context.Context, enrichmentContext enricher.EnrichmentContext, deployment *storage.Deployment) (*augmentedobjs.NetworkPoliciesApplied, error) {
+	storedPolicies, err := s.netpols.GetNetworkPolicies(ctx, enrichmentContext.ClusterID, enrichmentContext.Namespace)
+	if err != nil {
+		return nil, errox.InvalidArgs.New("failed to look up network policies for deployment").CausedBy(err)
```

Comment:
Is `InvalidArgs` reasonable here? Could `netpols/GetNetworkPolicies` return an error that is unrelated to the parameters passed?

Reply:
IMO this seems like a generic error, and we shouldn't try to fit into a generalized error box. Ideally, `GetNetworkPolicies` will return a error type based on the error that occured and we can just wrap with our message here. 

I suggest changing to:
```
errors.Wrapf(err, "failed to get network policies for cluster %s and namespace %s", cluster, namespace)
```

Reply:
Fair, changed!